editor.planx.uk/src/pages/Preview/ResumePage.tsx

```diff
@@ -178,6 +178,7 @@ const ResumePage: React.FC = () => {
   }, [email]);
 
   const teamSlug = useStore((state) => state.teamSlug);
+  const resumeSession = useStore((state) => state.resumeSession);
```

Comment:
I think `useStore` should only be called at the root of the component (see [React's rules of hooks](https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level))